### ISSUE TITLE
Link present on the video is visible as text.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-WebChat#1379]
**Actual Result:**
Link present on the video is visible as text.

**Expected Result:**
Link present on the video should be visible as link. Link should be implemented with proper underline.

**Repro Steps:**

1. Open URL [https://microsoft.github.io/BotFramework-WebChat/full-bundle/](https://microsoft.github.io/BotFramework-WebChat/full-bundle/) in Edge browser.
2. Navigate to 'Show a video attachment from Vimeo' button and activate it.
3. Video will appear at the bottom of the page.
4. Navigate the video with tab key and activate 'Share' button.
5. Observe if link present on the video is visible as link or not.

**User Impact:**
If any link is visible as text then visually impaired user will might get confuse about the purpose of that control.

**Test Environment:**
OS: Windows 10
OS Build: 17763.107
OS Version: 1809
Browser: Edge

**Attachment:**
![usable-link](https://user-images.githubusercontent.com/41668935/48763254-d4aa0680-ecd2-11e8-8a11-6af22bf73195.png)

**Comment by compulim** [comment]
This is not a bug from Web Chat. The video content is an IFRAME owned by Vimeo.

**Comment by compulim** [comment]
Reopening so we can confirm from the accessibility team.

**Comment by corinagum** [comment]
Confirmed with Accessibility team as External

**Comment by Amit8527510735** [comment]
#HCL;#Accessibility;#HCL_BotFramework_WebChat;A11yMAS;WCAG2.4.4
